src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
-<html lang="en">
+<html
+  class="page"
+  lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/index.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body
+   class="page__body">
```

Comment:
```suggestion
  <body class="page__body">
```
move each attribute on new line only if there are 3 or more attributes